changelog/fix16997.dd

```diff
@@ -0,0 +1,19 @@
+-preview=intpromote switch is now set by default.
+
+integer promotions now follow C integral promotions rules consistently
```

Comment:
Capital letter at the beginning of a sentence.

Reply:
Done